### ISSUE TITLE
Peformance and PWA categories cause failures when triggered from PHP

### ISSUE THREAD

**Issue by noyearzero** [GoogleChrome/lighthouse#7922]
I'm using PHP (Windows) to trigger lighthouse reports using the CLI in nodejs.  Everything works fine except if the performance and pwa categories are selected.  Running the exact same command directly in a command prompt works fine.  The user account PHP runs as is an administrator and the same user that can successfully run the command manually.  

I've tried various methods to trigger the audit and it always fails if PHP starts the process and performance or pwa is chosen.  For example:
PHP > command line with exec()
PHP > nodejs app > command line
PHP > nodejs app > lighthouse module

Also I've tried various versions of lighthouse.  3.2.1 and 4.2.0.

**Comment by Hoten** [comment]
What are the errors?

**Comment by noyearzero** [comment]
That's kind of tricky.  Since the problem only happens when chaining through multiple systems it's hard to see them.  When running PHP > command line there is no output with a response error code of 1.  When I ran the command line manually it worked, so again no error.  When running PHP > node app > command line, again no output and a response code of 1.  

I got the farthest with PHP > node app > lighthouse module.  I've gotten a promise rejection of:
name: LHError
code: NO_FCP
friendlyMessage: Something went wrong with recording the trace over your page load. Please run Lighthouse again. (NO_FCP)
lhrRuntimeError: 1

I'm not totally sure if that's the real error or something more with my coding (since I've had to figure out the node package only now because of the command line issues.

**Comment by Hoten** [comment]
Are you setting the out param in `exec` to get the output, or just using the return value (last line)?

Are you launching Chrome yourself or is LH handling that for you?

Using LH on this same machine, with the same user, within the same user session (ssh or whatever) with the same URL works, but fails when doing something like `php -r "exec('lighthouse https://www.example.com')"` (or w/e, typed that free-hand)?

If you get a minimal repro of this we can begin to help, but without that we probably can't figure this out for you.

**Comment by noyearzero** [comment]
The return value of the function call (the last line of the result of the command) is a blank string.  
The output param comes back with an empty array.
The return variable (usually 0 for success) is 1.  

When doing the node package method, I was using the provide example almost verbatim ... with require('chrome-launcher')

What you describe doesn't sound _exactly_ like what I'm doing.  Ultimately what I want to do is run the command `lighthouse https://www.example.com` - I need to trigger this from inside PHP because it is being triggered from a web based user interface (thus the need for exec()).  I don't need to trigger PHP itself from the command line.  typing the command `lighthouse https://www.example.com` directly in a command prompt works fine, but triggering that same command in exec() in a PHP script is where the problem is.  

I'm not sure how to setup a repro for this since there are so many moving parts.  I did see some other bugs that affected the performance and pwa categories recently.  It looks like they were fixed,  But maybe there is some other related issue that still manifests itself in very specific circumstances. I was hoping this would be enough to put someone on the right track.
